.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+
+Hey Contributor! :smiley_cat:
+
+Any contribution(s) to PHPSA are very much encouraged, and we do our best to make it as welcoming and simple as possible.
+
+## Coding Standards
+
+We require that all contributions meet at least the following guidelines:
+
+* PSR-1 & PSR-2
+* We are using camelCase for variables and methods/functions.
+* Don't use functions for casting like `intval`, `boolval` and etc, We are using `(int) $a`.
+* Avoid "Yoda conditions", where constants are placed first in comparisons:
+
+```php
+if (true == $someParameter) {
+}
+```
+
+* Avoid strict compare When it's not needed.
+* Avoid `static` methods.
+* Use `static::` binding if it's rly needed, mostly you will use `parent::` or `self::`
+* Don't use `Singleton` patern anywere.
+* Avoid aliases for functions: `sizeof`, `join` and etc.
+* Avoid global variables.
+* Use type stricts.
```

Comment:
"Use type stricts" → "Use strict types"

An example would be nice, I don't really get what you mean :)